drivers/weather/CMakeLists.txt

```diff
@@ -23,6 +23,14 @@ add_executable(indi_watcher_weather ${weatherwatcher_SRC})
 target_link_libraries(indi_watcher_weather indidriver ${CURL})
 install(TARGETS indi_watcher_weather RUNTIME DESTINATION bin)
 
+# ########## AAGSolo Weather Station ###############
+SET(aagsolo_SRC
+    aagsolo.cpp)
+
+add_executable(indi_aagsolo ${aagsolo_SRC})
```

Comment:
Traditionally, it should be indi_aagsolo_weather to be consistent with other INDI drivers.

Reply:
OK, will change that.